CHANGES.rst

```diff
@@ -102,6 +102,9 @@ New Features
     a ``Quantity`` angle or angular speed passed in to the argument ``x``.
     [#5653]
 
+  - Added a new equivalence between g/mol to atomic mass units.
```

Comment:
I thought @mhvk requested the equivalency name to be in the log. And I remember seeing you adding it. But seems lost now.